certbot/certbot/reverter.py

```diff
@@ -332,7 +337,7 @@ def register_file_creation(self, temporary, *files):
             if new_fd is not None:
                 new_fd.close()
 
-    def register_undo_command(self, temporary, command):
+    def register_undo_command(self, temporary: bool, command: List[str]) -> None:
```

Comment:
I guess we did agree to try use generic argument types in the `acme` PR:

```suggestion
    def register_undo_command(self, temporary: bool, command: Iterable[str]) -> None:
```

(+ import)

Reply:
Totally! By the way, when we need to represent an object that is iterable, sizable and with element access by index, we can use `typing.Sequence` instead of `typing.List` to get more genericity and allow tuples for instance.

However I am not sure to have the time to realign existing types in the module, anyway it would be done in a follow-up PR in my opinion.